### ISSUE TITLE
6/1 Laundry List

### ISSUE THREAD

**Issue by pringshia** [facebookresearch/Mephisto#158]
### Web UI

- [ ] Denote required fields in the UI when launching tasks
- [ ] Hide the mock architect in the UI since it is only used for testing purposes
- [ ] Should we hide the mock requester account as well? Should we add a local requester instead?
- [ ] Add a better file picker in the UI (but for this we need to have the back-end be able to denote an arg as type file)
- [x] A task failed because it was run with a local architect w/ an mturk requester. mTurk threw an error because the URL was localhost. This task however still shows up in the launch view. Perhaps change the task to a different state and expire all units?

### CLI

- [x] Make `mephisto check` auto-register the mock requester if one was created
- [x] Typecast registered (issue: Mock_REQUESTER was saying registered is 0 instead of False)
- [x] Mock requester seems to be unregisterable via `mock_requester.py`. Should we move the registering to `mephisto check`?
- [x] Was unable to force quit a task. Kept getting the message: `Waiting on 1 task runs, Ctrl-C ONCE to FORCE QUIT`. Repro: (blueprint: static_react_task, architect: local, requester: mock)

**Comment by JackUrb** [comment]
I'm closing this as being stale, it's referenced in another task at this point though.
